### PR TITLE
Feat: Reset password

### PR DESCRIPTION
**Pull request by OlgaSavina**
## Describe your changes

- I did pages for changing password and page of successful changing of password
restore
restore1

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://testhomework4.atlassian.net/jira/software/projects/AG/boards/6?selectedIssue=AG-108)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [ ] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [ ] Date and time formats are on the constants
- [ ] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

**Comment by anette1983** [comment]
approve

**Review by anette1983 — APPROVED** [review]

File: src/locales/langs/de.json [hunk 1]
```diff
@@ -144,6 +144,25 @@
...
+      "agree": "By continuing, you agree to AgentWise",
+      "and": "and"
+  },
+  "RestorePasswordDonePage":{
+    "title": "Password reset was successfull",
+    "text": "Your password has been successfully reset. Click below to log in magically",
+    "buttonText": "Log in"
   },
   "VerifyEmail": {
     "title": "Bitte überprüfen Sie Ihre E-Mails!",
```
Comment (AKutusenko) [inline src/locales/langs/de.json]: why english text in de.json?

Comment (OlgaSavina) [reply src/locales/langs/de.json]: We don't use german now as I know

Comment (lyudad) [reply src/locales/langs/de.json]: no, I suggest delete this file, to avoid confusion

**Comment by lyudad** [comment]
Screenshot 2023-11-24 at 20 48 32

File: src/locales/langs/de.json (hunk 1 above)
Comment (OlgaSavina) [reply src/locales/langs/de.json]: De.json is deleted now

**Pull request merged**